installer.sh

```diff
@@ -39,10 +40,23 @@ function finish {
     rm -rf $DIR/SDL2_mixer-2.0.0 || true
     rm -rf $DIR/glog || true
     rm -rf $DIR/yaml-cpp || true
+    rm -rf $DIR/astyle || true
 }
 
 trap finish EXIT
 
+#artistic style
```

Comment:
The reason this wasn't included was because this was more of a runtime installer script, not a developer one, like this one doesn't include installing cmake and stuff too. I think I would still like to keep this one as just a runtime install script.

I can see the motivation to automate installing astyle though, maybe that can be in a separate script, so for installing we keep a dev installer, runtime installer, and another one that installs for all (runs both the dev and runtime script). 

I do plan to dockerize the build process though, so at that maybe we don't even need a dev install script.  Thoughts?

Reply:
Kinda makes sense. Is there anything else thats non-dev ATM?

I was thinking this of more of a Kickstarter for devs, so we devs don't have to look for packages to download. So either we do one for runtime and then one for dev, or just put it all together (kickstart for devs) so we should include things like cmake and astyle too.

Reply:
Let's make a dev and runtime installer. Dev needs all the runtime libraries, so dev use the runtime installer for one of the steps.

Reply:
Want me to do that on this branch or as a separate pr

Reply:
I think for this PR removing the apt call is good, but not adding the astyle install. So for this PR let's just do the apt-get removal, then open a separate ticket for the dev dependencies.